packages/cdk/bin/cdk.ts

```diff
@@ -28,6 +28,12 @@ export const stacks: CloudwatchLogsManagementProps[] = [
 			'/aws/lambda/braze-exporter-callback-PROD',
 		],
 	},
+	{
+		stack: 'investigations',
```

Comment:
The `stack` value here is only used to generated this app's `riff-raff.yaml` file, and ultimately to decide which AWS account to deploy into.

IIUC the `investigations` stack and the `pfi` stack map to the same AWS account. This application deploys 3 lambdas. Targeting the same AWS account twice, we result in 6 lambdas in the account. Not sure this is desired?

Should we move the `/aws/lambda/transcription-service` value into the `logShippingPrefixes` of the `pfi` stack below, to avoid duplicating the lambdas?

Reply:
Thanks, great shout